client/packages/inventory/src/Stocktake/ListView/StocktakeFilters.tsx

```diff
@@ -0,0 +1,148 @@
+          Input={
+            <MasterListSearchInput
+              disabled={false}
+              onChange={masterList =>
+                setState(prev => ({
+                  ...prev,
+                  masterList,
+                }))
+              }
+              selectedMasterList={masterList}
+              width={380}
+              placeholder={t('label.all-items')}
+              clearable
+            />
+          }
+          label={t('label.master-list')}
+        />
+        <RadioGroup
+          value={includeAllItems}
+          sx={{
+            marginLeft: '160px',
+            display: masterList ? undefined : 'none',
+            transform: 'scale(0.9)',
+          }}
+          onChange={(_, value) => {
+            setState(prev => ({
+              ...prev,
+              includeAllItems: value === 'true',
+            }));
+          }}
+        >
+          <FormControlLabel
+            value={false}
+            control={<Radio sx={{ padding: '4px' }} />}
+            label={t('stocktake.items-with-soh')}
+          />
+          <FormControlLabel
+            disabled={!masterList || !!expiryDate || !!location || !!vvmStatus}
+            value={true}
+            control={<Radio sx={{ padding: '4px' }} />}
+            label={t('stocktake.all-master-list-items')}
+          />
+        </RadioGroup>
+      </Box>
+
+      <InputWithLabelRow
+        labelProps={{
+          sx: { flex: `${LABEL_FLEX}` },
+        }}
+        Input={
+          <LocationSearchInput
+            disabled={false}
+            onChange={location =>
+              setState(prev => ({
+                ...prev,
+                location,
+                includeAllItems: false,
+              }))
+            }
+            width={380}
+            selectedLocation={location}
+            placeholder={t('label.all-locations')}
+            clearable
+          />
+        }
+        label={t('label.location')}
+      />
+      <InputWithLabelRow
+        labelProps={{ sx: { flex: `${LABEL_FLEX}` } }}
+        Input={
+          <DateTimePickerInput
+            width={380}
+            value={expiryDate}
+            onChange={expiryDate =>
+              setState(prev => ({
+                ...prev,
+                expiryDate,
+                includeAllItems: false,
+              }))
+            }
+          />
+        }
+        label={t('label.items-expiring-before')}
+      />
+      {manageVvmStatusForStock && (
+        <InputWithLabelRow
+          label={t('label.vvm-status')}
+          labelProps={{ sx: { flex: `${LABEL_FLEX}` } }}
+          Input={
+            <VVMStatusSearchInput
+              onChange={vvmStatus =>
+                setState(prev => ({
+                  ...prev,
+                  vvmStatus: vvmStatus ?? null,
+                  includeAllItems: false,
+                }))
+              }
+              width={380}
+              selected={vvmStatus}
```

Comment:
clearable and placeholder was removed from here... Are we not allowing vvm status to be cleared in the stocktake filter modal?

Reply:
Pass? I'm guessing we should always have a VVM Status as part of a stocktake, since we have a history of vvm entries removing doesn't really make sense? That said havent' worked on this!

Reply:
Placeholder and clearable should be included - https://github.com/msupply-foundation/open-msupply/pull/9428/files#diff-044a34fbd6accdad0c15d3d659d9f398347578062b86695cb13a0cac47749d9fR142